homeassistant/components/emulated_hue/__init__.py

```diff
@@ -204,7 +205,9 @@ def __init__(self, hass, conf):
         # Get the IP address that will be passed to the Echo during discovery
         self.host_ip_addr = conf.get(CONF_HOST_IP)
         if self.host_ip_addr is None:
-            self.host_ip_addr = util.get_local_ip()
+            self.host_ip_addr = self.hass.loop.run_until_complete(
```

Comment:
The context switch can be avoided by moving this into `async_setup` and moving  `        await config.async_setup()`  from `start_emulated_hue_bridge` right after the Config object is created in `async_setup`

Reply:
Done